### DISCUSSION TITLE
Datadog plugin should emit service name and status code as tags

### DISCUSSION THREAD

**Discussion by judithpatudith** [Kong/kong#7268] [Ideas and feature requests]
Please read the CONTRIBUTING.md guidelines to learn on which channels you can
seek for help and ask general questions:

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-seek-for-help

### Summary

Issue reported by @irabinovitch

We noticed that  it is emitting metrics in the format:  kong.<service-name>.request.status.<status-code>

We generally would encourage service-name and status code to be tags rather than part of the metric name as it significantly improves the experience as one graphs/queries/alerts on the metric.  We've had a few customers reach out with similar feedback, so I wanted to circle back and see if it might be possible to make the change in an upcoming Kong release.

### Steps To Reproduce

1.
2.
3.
4.

### Additional Details & Logs

- Kong version (`$ kong version`)
- Kong debug-level startup logs (`$ kong start --vv`)
- Kong error logs (`<KONG_PREFIX>/logs/error.log`)
- Kong configuration (the output of a GET request to Kong's Admin port - see
  https://docs.konghq.com/latest/admin-api/#retrieve-node-information)
- Operating system

**Comment by irabinovitch** [comment]
Thanks @judithpatudith.  It looks like some of the variables might have gotten mangled by markdown.

The metrics we're seeing are `kong.<service-name>.request.status.<status-code>`.  Ideally it would be something like `kong.request.status` tagged by service name and status code.

**Comment by zhouzhuojie** [comment]
Hey Kong team, just to confirm, is this issue fixed in https://github.com/Kong/kong/pull/5154/files?
